charts/payment-integration/values.yaml

```diff
@@ -0,0 +1,28 @@
+image:
+  repository: ''  # either 'sphereio/commercetools-payone-integration' or 'commercetoolsps/commercetools-paypal-plus-integration/'
+  tag: ''  # set tag here
+  pullPolicy: IfNotPresent
+
+containerName: payment-integration
+
+service:
+  type: NodePort
+  externalPort: 80
+  internalPort: 8080
+
+rollingUpdateStrategy:
+  maxSurge: 1
+  maxUnavailable: 0
+
+portName: backend-port
+
+resources:
```

Comment:
resources requests/limits should be commented in the default `values.yaml` (as recommendation)

Reply:
a535228